### DISCUSSION TITLE
Is there any way to hide warnings in Rust's diagnostics panel?

### DISCUSSION THREAD

**Discussion by spencerkohan** [zed-industries/zed#6652] [Feature Requests]
It's just a bit noisy and it would be helpful to see errors only

**Comment by JosephTLyons** [comment]
This just landed in v0.106.1-pre:

https://zed.dev/releases/preview/0.106.1

**Reply by krstp** [reply]
Hmm... seems you guys do not preserve the ful release log. The oldest release notes are for `0.107.7` adn above link does not work :(
